src/index.js

```diff
@@ -25,10 +25,10 @@ export default class SaucelabsConnector {
         webDriver.once('status', () => {
             getSessionId()
                 .then(sid => {
-                    process.stdout.write('Browser started. See at https://saucelabs.com/tests/' + sid + '\n');
+                    process.stdout.write('Browser started. See https://saucelabs.com/tests/' + sid + '\n');
 
-                    // HACK: if the webDriver don't get any command during 1000s it's fail
-                    // with the timeout error. We should send any command to avoid this.
+                    // HACK: HACK: if the webDriver doesn't get any command within 1000s,
+                    // it fails with the timeout error. We should send any command to avoid this.

```

Comment:
Duplicated `HACK:` entry
And try to keep the comment strings with identical lengths. For example:
not:
```js
// HACK: if the webDriver doesn't get any command within 1000s,
// it fails with the timeout error. We should send any command to avoid this.
```
(the second string is more longer)
but:
```js
// HACK: if the webDriver doesn't get any command within 1000s, it fails 
// with the timeout error. We should send any command to avoid this.
```
(the strings have almost equal length)